en/components/grid-editing.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Grid Editing - Grid Feature
+_description: The Grid Editing allows the user to update the values of the records displayed in the Grid.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+## Grid Editing
+
+Use Grid Editing to allow the user to update certain values either cell by cell, or one row at a time. Both Cell and Row Editing are visually identical to the [Ignite UI for Angular Grid Editing Feature](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/editing.html)
+
+### Cell Editing
+
+The Grid Cell Editing happens cell by cell and is configured via the Body Cell that one want to display in edit mode via the `Grid Feature` override. Once you set it to `Editing/Cell`, you will have an editable Grid that respects this mode.
+
+<img class="responsive-img" src="../images/grid_cell_edit.png" srcset="../images/grid_cell_edit@2x.png 2x" />
+
+### Row Editing
+
+The Grid Row Editing happens one row at a time and is configured via the Body Cells that belong to the edited row by setting their `Grid Feature` override to `RowEditing/Cell`. This will introduce a second override `Editing State`allowing you to specify the cell in focus by setting it to `Cell:focus`, bear in mund that you should set all other cells on this row to `Cell:no focus`.
+
+<img class="responsive-img" src="../images/grid_row_edit.png" srcset="../images/grid_row_edit@2x.png 2x" />
+
+### Editing Row Banner
+
+The Grid has an `Editing Row Banner` symbol under `Features` that represents a summary area with the number of edits made on the row and a pair of actions for cancelling and confirming the edits made on the row. Adding this symbol has only a visual effect on your design to make it appear more realistic, if you want to switch the Row Editing feature, configure the Cells of the row accordingly.
+
+## Code generation
+
+If the `Editing Row Banner` symbol is added make sure to place it within the group defining the Grid as a sibling to the Header and Body groups.
```

Comment:
Similar to my other comments, the editing row banner doesn't enable the feature on the grid.  It is the override on the Header Cell that enables it since it has the option for Row Editing and regular Cell Editing.  So we should mention it here in the code gen section.

Reply:
And again the paragraph above explains exactly that: The Grid Row Editing happens one row at a time and is configured via the Body Cells that belong to the edited row by setting their `Grid Feature` override to `RowEditing/Cell`. This will introduce a second override `Editing State`allowing you to specify the cell in focus by setting it to `Cell:focus`, bear in mund that you should set all other cells on this row to `Cell:no focus`.

Reply:
It's probably better if I change codegen so that it enables row editing on the grid if the row editing banner is present.  I'll make it so that is the case.

However I want to make it clear that we don't determine whether a column is editable based on the body cell.  We only look at the Header Cell's `Editing` override.  If this override is set to `Editable` or `Row Editable` then we enable editing on the column.

Reply:
That sounds good with me.